berkeley-mobile/Resources/Campus/ResourceDetailViewController.swift

```diff
@@ -104,9 +104,9 @@ extension ResourceDetailViewController {
 
 // MARK: - Analytics
 
-extension ResourceDetailViewController {
+extension CampusResourceDetailViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        Analytics.logEvent("opened_resource", parameters: ["resource" : resource.name])
+        Analytics.logEvent("opened_campus_resource", parameters: ["resource" : resource.name])
```

Comment:
We should probably keep this as is, otherwise the events on Firebase would get split up (between Android + old iOS users). You could add a thing to `CovidResourceViewController` though, so we could start tracking when people open the new tab!